Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
+    commander (4.5.2)
```

Comment:
### [commander](https://github.com/commander-rb/commander)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.4.7 → 4.5.2

[_[change-log](https://github.com/commander-rb/commander/blob/master/History.rdoc), [source-code](https://github.com/commander-rb/commander/tree/v4.5.2)_]





<details>
<summary>Commits</summary>

A change of **24** commits. See the full changes on [the compare page](https://github.com/commander-rb/commander/compare/v4.4.7...v4.5.2).

These are the first 10 commits:
- (73ea8e7) [Update list of Rubies for Travis](https://github.com/commander-rb/commander/commit/73ea8e79dac0c6aa1f0e9dcd5b3373251875ce72)
- (ec983df) [Drop support for Ruby < 2.3](https://github.com/commander-rb/commander/commit/ec983dfc2404080e143851d9c62c0f1cb625c8bf)
- (0a40045) [Merge pull request #82 from commander-rb/test-new-rubies](https://github.com/commander-rb/commander/commit/0a40045dabf0e041ac6133f8a1334f495e669b06)
- (b5ccddd) [Fix global option code to preserve double dash](https://github.com/commander-rb/commander/commit/b5ccdddfc928bbbc2b813e581299ad3cb9336cb4)
- (1e03bae) [Merge pull request #83 from commander-rb/fix-double-dash](https://github.com/commander-rb/commander/commit/1e03bae87cc8d9c905012d31e22b4fd1d149ec12)
- (9cf5f10) [Version 4.5.0](https://github.com/commander-rb/commander/commit/9cf5f10470baee4c380f0beb9b757ce71e071e0f)
- (bd878be) [README: Use https links](https://github.com/commander-rb/commander/commit/bd878be5671b70a132640eac76328d523f456789)
- (adaa197) [Merge pull request #85 from olleolleolle/patch-1](https://github.com/commander-rb/commander/commit/adaa197613bfc7833f7d3b23bc702a0507b9ee24)
- (b0ed230) [Add project metadata to the gemspec](https://github.com/commander-rb/commander/commit/b0ed230a339ec5242111c0769a9b8bb92c8a3e0d)
- (1cf9738) [Fix: Global option values ignored when args provided](https://github.com/commander-rb/commander/commit/1cf973855be4fc183dc7e92fecc5262b95d3a2cf)


</details>